src/index.html

```diff
@@ -13,6 +13,54 @@
     />
   </head>
   <body>
-    <h1>Stars</h1>
+    <div class="star__blok">
```

Comment:
The class name `star__blok` should be corrected to `star__block` to follow proper English spelling and BEM conventions.